app/src/main/java/chat/rocket/android/contacts/models/Contact.kt

```diff
@@ -100,4 +120,13 @@ class Contact() : Parcelable {
         }
     }
 
-}
\ No newline at end of file
+    interface CARD_TYPE {
+        companion object {
+            val VIEW_PUBLIC_ACCOUNT= 0
+            val VIEW_CONTACT = 1
+            val VIEW_HEADING= 2
+            val VIEW_INVITE_OTHER_APP= 3
```

Comment:
As per my other comment, it seems odd that we would have to overload the Contact object in order to handle these other cases.

Reply:
Removed